static/sass/styles.scss

```diff
@@ -1366,20 +1366,6 @@ $color-link-dark: #69c !default;
   }
 }
 
-// XXX: @bartaz: this will be covered in Vanilla
-// can be removed once https://github.com/canonical/vanilla-framework/issues/4875 is done and released
-.p-logo-section.has-misaligned-images {
```

Comment:
I haven't finished qa'ing all the pages that use this, but it looks like the logo sections in https://ubuntu-com-13548.demos.haus/financial-services still looks buggy without it compare to prod

Reply:
Also, this seems to just remove the comment, but not actual code. Is this intended?